src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -100,6 +100,10 @@
      * @param allFields: parsed list of all possible fields for dueling picklist
      */
     loadAvailableFields: function(component, activeFields, allFields) {
+        const FIELD_SOBJECT_TYPE_OPPORTUNITY = 'opportunity';
```

Comment:
It looks like we don't use this "FIELD_SOBJECT_TYPE_OPPORTUNITY" anywhere, so can we pull it out?

Reply:
Yes we can, I will aso remove "opportunityFields" and "paymentFields"